samples/test/list_testcase-results.test.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+const execSync = require('child_process').execSync;
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+const dialogflow = require('@google-cloud/dialogflow-cx');
+
+describe('Test filtering results', async () => {
+  const cmd = 'node list-testcase-results.js';
+  const agentId = 'be66a5a2-4602-4f8c-a3f3-046896904362';
```

Comment:
What will happen if these tests run in parallel?  Guessing we should be using a generated GUID and not a hard coded one?

Reply:
This test should still run properly even when it runs in parallel, i used hardcoded agent id to simplify testing and if i created the agent and necessary components to run this test it would increase code that could fail due to quota limits.

Reply:
Perhaps compromise and pull it into an environment variable, so that this is a bit more configurable?

You can use a `pre-sample-test.sh` file to do so, see https://github.com/googleapis/nodejs-storage/blob/main/.kokoro/pre-samples-test.sh